docs/tutorial2023_unfolding/unfolding_exercise.md

```diff
@@ -0,0 +1,251 @@
+
+```
+Now we can plot the likelihood scan and extract the expected intervals.
+
+```shell
+python scripts/plot1DScan.py higgsCombinescan_r_zh_75_150.MultiDimFit.mH120.root -o r_zh_75_150 
+```
+> Repeat for all POIs
+
+## Shape analysis with control regions
+
+One of the advantages of the maximum likelihood unfolding is the flexibility to choose the analysis observable and include more information on the event kinematics, consequently improving the analysis sensitivity. This analysis benefits from the shape information of the DNN output trained to differentiate the VH(bb) signal from the SM backgrounds. 
+
+The datacards for this part of the exercise located `full_model_datacards/`, where you can find a separate datacard for each region within `full_model_datacards/regions` directory and also a combined datacard `full_model_datacards/comb_full_model.txt`. 
+ 
+As you will find the datacards also contain several background processes. To control them properly we will add the regions enriched in the respective backgrounds. Then we can define a common set rate parameters for signal and control regions to scale the rates or other parameters affecting their shape.  
+
+For the shape datacards one has to specify the mapping of histograms and channels/processes as given described below:
+
+```
+shapes [process] [channel] [file] [nominal] [systematics_templates]
+```
+Then the `shape` nuisance parameters can be defined in the systematics block in the datacard. 
+
+In the realistic CMS analysis there are hundreds of nuisance parameters corresponding to various source of systematics. 
+
+When we unfold to the gen-level observable we should remove the nuisances affecting the rate of the gen-level bins, i.e. the `lnN` NPs: `THU_ZH_mig*, THU_ZH_inc` and keep only the acceptance `shape` uncertainties: `THU_ZH_acc` and `THU_ggZH_acc`. This can be achieved by freezing the respective nuisance parameters with the option `--freezeParameters par_name1,par_name2`. Alternatively you can create a group following the syntax given below at the end of the combined datacard, and freeze the parameters with the `--freezeNuisanceGroups group_name` option.
+
+```
+[group_name] group = uncertainty_1 uncertainty_2 ... uncertainty_N
+```
+
+Now we can create the workspace using the same `multiSignalmodel` 
+
+```shell
+text2workspace.py -m 125  full_model_datacards/comb_full_model.txt -P HiggsAnalysis.CombinedLimit.PhysicsModel:multiSignalModel  --PO verbose --PO 'map=.*/.*ZH_lep_PTV_75_150_hbb:r_zh_75_150[1,-5,5]' --PO 'map=.*/.*ZH_lep_PTV_150_250_0J_hbb:r_zh_150_250noj[1,-5,5]'  --PO 'map=.*/.*ZH_lep_PTV_150_250_GE1J_hbb:r_zh_150_250wj[1,-5,5]' --PO 'map=.*/.*ZH_lep_PTV_250_400_hbb:r_zh_250_400[1,-5,5]' --PO 'map=.*/.*ZH_lep_PTV_GT400_hbb:r_zh_gt400[1,-5,5]' --for-fits --no-wrappers --X-pack-asympows --optimize-si.png-constraints=cms --use-histsum -o ws_full.root
```

Comment:
I think `-simpdf-` has morphed into `-si.png-` in the `--optimize-simpdf-constraints` flag. It might also be good to explain somewhere a little what some of these flags do? Perhaps we could add a collapsable info box, or similar?